### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_New Database_I want to use the new database dialog to create new database_Screen Reader: Voice over is not announcing the name of the "New Database" menu popup.

### ISSUE THREAD

**Issue by Raghu-addepalli** [microsoft/azuredatastudio-mysql#198]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #GH-MySQLextensionforAzureDataStudio-MAC-Mar23; #MySQL extension for Azure Data Studio; #Screenreader; #VoiceOver; #Benchmark; #WCAG4.1.2; #Name Role Value; #Benchmark; #FTP; #MAC; #AILimited; #A11yWCAG2.1; #Closed;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Application Name: Azure Data Studio
Application Version: 14.2.0
Operating system: macOS Ventura (Version 13.3)
Screen Reader: Voiceover.

### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till new connection and press enter
4. Enter a valid server name username and password and press connect

### Repro Steps:

1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server connection. 
4. Then select 'Manage' from the context menu to view the dashboard.
5. Turn on Voice over.
6. From dashboard, click on New Database.
7. Now check that Voice over is not announcing the name of the "New Database" menu popup is appeared or not.
### Actual Result:

Voice over is not announcing the name of the "New Database" menu popup when it appears.
### Expected Result:
Voice over should announce the name of the "New Database" menu popup when it appears.
### User Impact:
Screen Reader Users will not be able to get which menu popup has opened if the name of the menu popup is announced by Voiceover. 
### Attachment:
[s5.zip](https://github.com/microsoft/azuredatastudio-mysql/files/11121914/s5.zip)

**Comment by rishky-msft** [comment]
Hi @Raghu-addepalli ,

I have fixed the bug in attached PR.

You can download the pre-release extension and test it out.
https://github.com/microsoft/azuredatastudio-mysql/releases/download/v0.2.9-alpha.1/azuredatastudio-mysql-0.2.9-alpha.1.vsix

Steps for installing pre-release extension:
1. Uninstall the MySQL Extension if already installed. (Open ADS -> Extensions -> MySQL -> uninstall -> Restart ADS)
2.  Download the pre-release extension vsix from above link.
3.  Open ADS -> File -> Install Extension via VSIX File -> Select the dowloaded vsix.
4.  Pre release extension will be installed.

**Comment by Raghu-addepalli** [comment]
verified the issue by following above steps, now issue is not reproducing hence closing the bug and added attachment below.

https://user-images.githubusercontent.com/91586743/231135921-78bb763b-4b6c-43c9-8a61-219c36736255.mp4
